service/server.py

```diff
@@ -67,21 +68,38 @@ def get_properties_for_address(address):
     return query.execute().hits
 
 
+def paginated_address_records(address_records, page_number):
+    start_index = (page_number-1)*int(SEARCH_RESULTS_PER_PAGE)
```

Comment:
We could convert the 'search results per page' setting to int in the config module - this way the conversion would just happen once. Also, if the related environment property was invalid, the application would fail to start.

I'm also happy if it stays like this.